osbs/api.py

```diff
@@ -268,6 +268,18 @@ def _create_scratch_build(self, build_request):
             build_json['spec'] = {}
         build_json['spec']['serviceAccount'] = 'builder'
         build_json['metadata']['labels']['scratch'] = 'true'
+
+        builder_img = build_json['spec']['strategy']['customStrategy']['from']
+        kind = builder_img['kind']
+        if kind == 'ImageStreamTag':
+            # Only BuildConfigs get to specify an ImageStreamTag. When
+            # creating Builds directly we need to specify a
+            # DockerImage.
+            response = self.get_image_stream_tag(builder_img['name'])
+            ref = response.json()['image']['dockerImageReference']
+            builder_img['kind'] = 'DockerImage'
```

Comment:
I'm wondering if we should just delete `builder_img['kind']` since that attribute is not set when using an image reference.